src/app/user/user-login/user-login.component.scss

```diff
@@ -0,0 +1,87 @@
+@import 'src/styles.scss';
```

Comment:
`styles.scss` are the styles in the global browser scope. It would be a bit better to put the colors in a `colors.scss` and import only these to avoid pulling in many other CSS rules.